react-frontend/peerly.env.local

```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_CLIENT_ID = "xxxxx"
```

Comment:
dont commit .env files https://dev.to/somedood/please-dont-commit-env-3o9h

Reply:
also dont we have a proper google app setup for this? is this the dummy client id?